### PR TITLE
Use honcho/gunicorn for local dev server

### PR DESCRIPTION
**Pull request by joshfriend**

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -1 +1 @@
-web: gunicorn --env CONFIG=dev manage:app --log-file -
+web: gunicorn --reload --env CONFIG=dev manage:app --log-file -
```
Comment (jacebrowning) [inline Procfile]: Delete `CONFIG=dev` here?

File: memegen/.env [hunk 2]
```diff
@@ -0,0 +1 @@
+CONFIG="dev"
```
Comment (jacebrowning) [inline memegen/.env]: BRO do you even diff?

File: .env [hunk 3]
```diff
@@ -0,0 +1,2 @@
+GOOGLE_ANALYTICS_TID="UA-12345-AB"
```
Comment (jacebrowning) [inline .env]: I don't think this line should be committed.

File: memegen/.env (hunk 2 above)
Comment (joshfriend) [inline memegen/.env]: iw

**Pull request merged**